xwiki-commons-core/xwiki-commons-xml/src/test/java/org/xwiki/xml/internal/html/DefaultHTMLCleanerTest.java

```diff
@@ -125,7 +125,9 @@ void specialCharacters()
         assertHTML("<p>\"&amp;</p>", "<p>\"&</p>");
         assertHTML("<p><img src=\"http://host.com/a.gif?a=foo&amp;b=bar\" /></p>",
             "<img src=\"http://host.com/a.gif?a=foo&amp;b=bar\" />");
-        assertHTML("<p>&#xA;</p>", "<p>&#xA;</p>");
```

Comment:
Why remove that test ?

Reply:
I think we have other assertions checking this, but I'll check.

Reply:
Actually the assertion has not been removed it's the line just after: `assertHTML("<p>\n</p>", "<p>&#xA;</p>");`

Reply:
No, it's not the same thing IMO.

Reply:
Well the assertion checks that `<p>&#xA;</p>` is transformed to `<p>\n</p>`. Not sure what else we can do here?

Reply:
Yeah I first did same mistake...